### PR TITLE
Refactored employee page.

### PR DESCRIPTION
**Pull request by ScorusMihai**

### CODE REVIEW

**Review by CaprarClaudiu — CHANGES_REQUESTED** [review]
There are variables created inside the method, elements can be created/reused and too many wait times.

File: src/main/java/driver/DriverFactory.java [hunk 1]
```diff
@@ -239,4 +267,31 @@ public void openAddLicensesPage() {
...
+        if (roleAfterEdit.equals("user")) {
+            elementToLoad(userRoleButton);
+            userRoleButton.click();
+        } else if (roleAfterEdit.equals("super_admin")) {
+            elementToLoad(superAdminRoleButton);
+            superAdminRoleButton.click();
+        } else if (roleAfterEdit.equals("admin")) {
+            elementToLoad(adminRoleButton);
+            adminRoleButton.click();
+        }
```
Comment (CaprarClaudiu) [inline src/main/java/driver/DriverFactory.java]: this can be done easier using a switch case

File: src/main/java/pageObjects/EmployeePage.java [hunk 2]
```diff
@@ -35,62 +35,29 @@ public EmployeePage(WebDriver driver) {
     @FindBy(id = "user-saveBtn")
     WebElement saveButton;
 
+    @FindBy(id = "undefined-table")
+    WebElement table;
+
+    @FindBy(id = "success_toaster")
+    WebElement successToaster;
```
Comment (CaprarClaudiu) [inline src/main/java/pageObjects/EmployeePage.java]: there seem to be some unused web elements

Comment (CaprarClaudiu) [reply src/main/java/pageObjects/EmployeePage.java]: and imports

File: src/main/java/driver/DriverFactory.java [hunk 3]
```diff
@@ -96,6 +98,31 @@ public DriverFactory(WebDriver driver) {
...
+
+    @FindBy(id = "user-saveBtn")
+    WebElement saveButton;
+
+
+    @FindBy(id = "success_toaster")
+    WebElement successToaster;
+
+    private String username;
+    private String email;
```
Comment (CaprarClaudiu) [inline src/main/java/driver/DriverFactory.java]: if we use these strings only for one method, they can be declared in the method

File: src/test/java/pageObjects/EmployeePageTests.java [hunk 4]
```diff
@@ -4,27 +4,26 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import static pageObjects.EmployeePage.elementText;
```
Comment (CaprarClaudiu) [inline src/test/java/pageObjects/EmployeePageTests.java:6]: there is an unused import

**Review by CaprarClaudiu — CHANGES_REQUESTED** [review]

**Review by BiancaJoca — APPROVED** [review]

**Review by CaprarClaudiu — APPROVED** [review]

**Pull request merged**